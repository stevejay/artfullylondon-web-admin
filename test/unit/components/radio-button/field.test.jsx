import React from 'react'
import { shallow } from 'enzyme'
import _ from 'lodash'

import RadioButtonField from '_admin/components/radio-button/field'

it('should render correctly', () => {
  const wrapper = shallow(
    <RadioButtonField
      label='The Label'
      input={{
        value: 'id-1',
        name: 'The Name',
        onChange: _.noop
      }}
      options={[
        { id: 'id-1', label: 'Label One' },
        { id: 'id-2', label: 'Label Two' }
      ]}
    />
  )

  expect(wrapper).toMatchSnapshot()
})
