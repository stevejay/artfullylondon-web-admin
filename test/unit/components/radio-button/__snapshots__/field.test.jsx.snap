// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<Container
  disabled={false}
  htmlFor="The Label"
  label="The Label"
  required={false}
>
  <div
    className="components-radio-button-field-m__options-container"
  >
    <RadioButton
      checked={true}
      key="id-1"
      label="Label One"
      name="The Name"
      onChange={[Function]}
      value="id-1"
    />
    <RadioButton
      checked={false}
      key="id-2"
      label="Label Two"
      name="The Name"
      onChange={[Function]}
      value="id-2"
    />
  </div>
</Container>
`;
